abstract_specification.md

```diff
@@ -649,6 +655,12 @@ Examples, using a pseudo-syntax:
 
 ## Platform-Specific Configuration
 
+SOMA includes provisions for two separate ways to provide platform-specific configuration data to objects and operations.
+For configuration settings that affect a single operation, SOMA functions include a `platform_config` parameter, which can be used on a call-by-call basis to specify short-term behavior.
```

Comment:
I find the `short-term behavior` phrase confusing, as some of the `platform_config` parameters modify schema, which has long-term effects.

Suggest using `... to specify per-operation behavior`, or even more explicitly, `... to specify behavior of a single operation call.`

Reply:
I've tried to rephrase this.